server/server_code/database.py

```diff
@@ -14,5 +27,58 @@ def get_users(self):
   
```

Comment:
Does the above need to be CAPS SELECT as well?

Reply:
Hey Shane, what do you mean by CAPS SELECT?